CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## v0.3.1 [2022-04-22]
+
+_Bug fixes_
+
+- Fixed the invalid page size error in the following tables when limit passed is more than 999: ([#50](https://github.com/turbot/steampipe-plugin-azuread/pull/50))
```

Comment:
- Fixed the following tables' results being limited to 999 rows unintentionally: (([#50](https://github.com/turbot/steampipe-plugin-azuread/pull/50)))